shared/chat/inbox/row/big-team-channel/container.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+import * as Selectors from '../selectors'
+import * as Creators from '../../../../actions/chat/creators'
+import {BigTeamChannel} from '.'
+import {pausableConnect, type TypedState} from '../../../../util/container'
+
+const mapStateToProps = (state: TypedState, {conversationIDKey, channelname}) => {
+  const p = Selectors.snippetRowSelector(state, conversationIDKey)
+  return {
+    channelname,
+    hasBadge: p.hasBadge,
+    hasUnread: p.hasUnread,
+    isMuted: p.isMuted,
+    isSelected: p.isSelected,
+    showBold: p.showBold,
+  }
+}
+
+const mapDispatchToProps = (dispatch: Dispatch, {conversationIDKey}) => ({
+  onSelectConversation: () => {
+    dispatch(Creators.setInboxFilter(''))
+    dispatch(Creators.selectConversation(conversationIDKey, true))
+  },
+})
+
+const mergeProps = (stateProps, dispatchProps, ownProps) => ({
```

Comment:
This and other connectors use explicit mergeProps always so we don't have any OwnProps (etc) that will leak and cause a re-render of the component

Reply:
Perhaps worth making BigTeamChannel's Props an exact type? (And similarly for the other related classes.) I tried it, and you'll have to add `isActiveRoute` to it and clean up all the uses in `index.stores.js`, but seems like it would automatically enforce this.

Reply:
hmmm, lets chat about this. `isActiveRoute` is actually only consumed by the pausableConnect hoc around this component , and right now it passes that prop down into the component but thats more of an implementation detail. If a non-route child needs this info and has inner pausableConnected components then it needs this, but otherwise it doesn't. So BigTeamChannel doesn't actually need or use this prop so adding it to the storyboard is actually not ideal. maybe we need 2 types of pausableConnect, one which passes and one which doesn't. i'll ticket it to discuss

Reply:
Yeah, having to add `isActiveRoute` to the component isn't great, but I thought having the exact type would be worth it. That was more of a suggestion, though, this seems important enough to go in as-is, and we can maybe fix it up later.